docs/prover-submission-cost/prd.md

```diff
@@ -0,0 +1,134 @@
+
+The BaseFee calculation originally assumed sufficient mana utilization per block to meet targets. Therefore, blocks with low utilization would not fully cover operational costs through fees alone. To address this, block rewards were introduced as a subsidy.
+
+Provers were compensated with a share of both fees and rewards, allocated according to their provided quotes.
+
+Then the move to [Prover Coordination: Multiproofs](https://hackmd.io/Ivn9axP1SFyEHjpAXVn62g) happened and the costs morphed into the following (`M` is number of proofs):
+
+1. the cost to prove the transactions in the block `M` times
+2. the cost to submit the block
+3. `M/N` cost to submit an epoch proof
+
+However, our model is currently flawed as it is:
+
+- not taking `M` proofs into account when computing the BaseFee
+- paying the proof submission cost to the sequencer, not the prover
+
+These flaws are remnants of the quote-based prover coordination.
+
+## Assumptions
+
+- There exists at least `M` provers that are willing to participate in block production.
+  - If the true number is less than `M` we are overpaying for proofs.
+- The gas estimates of the proof submission cost is somewhat precise
+  - Even though it happens potentially many blocks before the proof submission itself.
+- We expect submission cost to be greater than proving cost.
+
+# User Stories
+
+## End User
+
+As an end user, I want transaction fees to be predictable, transparent, and low, similar to using a single-sequencer system, so that I don’t face unexpectedly high costs.
+
+This is especially important during early network phases with low usage, when high fees might discourage me from continuing to use the network.
+
+> [!WARNING]
+> **Multi-proof overhead**   
+> If the costs associated with submitting multiple proofs (M) are passed directly to users via the BaseFee, it may lead to significantly higher fees compared to centralized or single-proof setups. The additional application of the Congestion Multiplier on these inflated fees could further compound this issue, potentially causing users to abandon the network early on.
+
+## Prover
+
+As an economically rational prover, I wish to participate in block production on the Aztec network, such that I can earn some money on all the machines that I have collected.
+
+In short, as a prover, I wish to:
+
+1. Collect blocks and transactions as the chain grows
+2. Run machines to prove transactions and roll them into an epoch proof
+3. Submit epoch proofs to the base-layer for a share of the rewards
+4. Profit???
+
+> [!WARNING]
+> **Multi-proof commentary**   
+> With the nature of the multi-proofs splitting rewards between all submitters and only paying out after. We won't know ahead of time if there is profit or not.
+
+# Requirements
+
+## Functional Requirements
+
+### Earmarked Fees
+
+- **What**: Fees earmarked for specific actions (e.g., proof submission) **MUST** be directed to the entity incurring the associated cost.
```

Comment:
Does this mean the sequencer, or governance dictates who receives the earmarked fee for proving and submission?

Reply:
i.e is M set by governance?